lib/pbkit/pbkit.c

```diff
@@ -2260,7 +2261,16 @@ void pb_end(DWORD *pEnd)
 #endif
 
 #ifdef DBG
-    if (pb_BeginEndPair==0) debugPrint("pb_end without a pb_start\n");
+    if (pEnd!=pb_PushNext)
+    {
+        debugPrint("pb_end: input pointer invalid or not following previous write addresses\n");
+        assert(false);
```

Comment:
Compilation issue: `error: use of undeclared identifier 'false';`

I can't stress this enough:
- At the very least compile each build before pushing
- Also test your builds regularly

These repeated compilation issues cause a lot of delays in review.

Reply:
`<stdbool.h>` seems to have been added now, but it's included even if `DBG` ist not defined (when it wouldn't be necessary). If we respect it for `assert.h`, we should also respect it for `stdbool.h`. It should just be consistent - I'm also fine with ignoring `DBG` for both of these paticular includes (having them regardless doesn't have any disadvantages).

Reply:
I opted for putting them both inside the `#ifdef` guard.